model/src/form/form-metadata/index.ts

```diff
@@ -41,6 +41,33 @@ export const privacyNoticeUrlSchema = Joi.string()
   })
   .trim()
 
+export const phoneSchema = Joi.string().trim()
+
+export const emailAddressSchema = Joi.string().email().trim().required()
+export const emailResponseTimeSchema = Joi.string().trim().required()
+export const emailSchema = Joi.object().keys({
+  address: emailAddressSchema,
+  responseTime: emailResponseTimeSchema
+})
+
+export const onlineLinkUrlSchema = Joi.string()
+  .uri({
+    scheme: ['http', 'https']
+  })
+  .trim()
+  .required()
+export const onlineLinkTextSchema = Joi.string().trim().required()
+export const onlineSchema = Joi.object().keys({
+  url: onlineLinkUrlSchema,
+  text: onlineLinkTextSchema
+})
+
+export const contactSchema = Joi.object().keys({
```

Comment:
Should we pass the expected types into Joi?

It's not the most advanced feature, but it'll shout at us if we miss new keys in future etc

```suggestion
export const contactSchema = Joi.object<FormMetadataContact>().keys({
```

Reply:
Resolved 68b3c9ec2f7f8c6b4775cf740ee46f4716d14792

Reply:
Happy to leave the other objects for now?

Reply:
For the child schema? c22fd5e999a68152ec20ecd73350eca0245a0232

Reply:
Cheers, think you've got them all now